### DISCUSSION TITLE
BLURRY EYES AND MOUTH IN LIVE MODE.

### DISCUSSION THREAD

**Discussion by 0769751170** [hacksider/Deep-Live-Cam#609] [Q&A]
HOW CAN I SOLVE BLURRY EYES AND MOUTH ON LIVE MODE

**Comment by hacksider** [comment]
https://github.com/hacksider/Deep-Live-Cam/wiki/Tips-on-how-to-improve-the-quality

**Reply by 0769751170** [reply]
I don't see that Nvidia Super Resolution filter which OBS version are you using?

**Comment by hacksider** [comment]
https://obsproject.com/forum/resources/obs-rtx-superresolution.1890/
